WooCommerce/Classes/ViewRelated/Products/Edit Product/BottomSheetListSelector/ProductTypeBottomSheetListSelectorCommand.swift

```diff
@@ -89,6 +101,12 @@ public enum BottomSheetProductType: Hashable {
         case .affiliate:
             return NSLocalizedString("Link a product to an external website",
                                      comment: "Description of the Action sheet option when the user wants to change the Product type to external product")
+        case .subscription:
+            return NSLocalizedString("A unique product subscription that enables recurring payments",
+                                     comment: "Description of the Action sheet option when the user wants to change the Product type to Subscription product")
+        case .variableSubscription:
+            return NSLocalizedString("A product subscription with variations",
+                                     comment: "Action sheet option when the user wants to change the Product type to Variable subscription product")
```

Comment:
As above

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/11118/commits/21ec2e8f13308e9780e4533184e3e08a390831ce